app.py

```diff
@@ -1,19 +1,27 @@
 import json
-
 from flask import Flask, request
-
 from flask_api import status
 from src.main.tag_identifier import TagIdentifier
 from src.main.tagging_algos.tagging_enums.optimization_tool_mapping import OptimizationToolMapping
 from src.main.utils.logger_initializer import *
 app = Flask(__name__)
 PARENT_DIR = os.getcwd()
 initialize_logger(PARENT_DIR)
+hostname = os.popen('hostname').read()
+# If on docker or local log to logs directory, if on AWS ec2 log to logs folder outside project
+if 'ec2' in hostname:
+    initialize_logger('../')
+else:
+    initialize_logger(PARENT_DIR)
 
 @app.route('/actuator')
 def actuator():
     logging.info("actuator")
-    return "success", status.HTTP_200_OK
+    return "success!", status.HTTP_200_OK
+
+@app.route('/')
+def healthcheck():
+    return "healthcheck", status.HTTP_200_OK
```

Comment:
whats the difference between actuator and healthcheck?

Reply:
nothing really, can remove the actuator endpoint or provide some sort of helpful information with it similar to the Java /actuator

Healthcheck is just whats getting hit by AWS LB

Reply:
got it. yeah probably remove for now if its unused